docs/products/kafka/howto/monitor-logs-acl-failure.rst

```diff
@@ -0,0 +1,42 @@
+Monitor and alert logs for denied ACL
+=====================================
+
+Aiven for Apache Kafka supports the definition of users and access control lists (ACL) to allow and limit producer and consumer rights at topic level. You read more about ACLs permission in the :doc:`dedicated documentation <../concepts/acl>`.
+
+In cases of ACLs problems, a ``io.aiven.kafka.auth.AivenAclAuthorizer`` is generated. 
```

Comment:
is ``io.aiven.kafka.auth.AivenAclAuthorizer`` an error? or message? would sound nicer if we specify similar to "In cases of ACLs problems, an error ``io.aiven.kafka.auth.AivenAclAuthorizer`` is generated"